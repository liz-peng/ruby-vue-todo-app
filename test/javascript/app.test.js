import { shallowMount } from '@vue/test-utils';
import App from 'app'; 
let wrapper;

beforeEach(() => {
    wrapper = shallowMount(App, {
        propsData: {},
        mocks: {},
        stubs: {},
        methods: {},
    });
});

afterEach(() => {
    wrapper.destroy();
});

describe('App', () => {
    test('is a Vue instance', () => {
        expect(wrapper.isVueInstance).toBeTruthy();
    });
});