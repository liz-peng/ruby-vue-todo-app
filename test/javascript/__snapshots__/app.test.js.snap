// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<div id="app">
  <p>Hello Vue!</p>
</div>
`;
